src/Pages/Ootd/Main/CardList.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import Card from "./Card";
+import './CardList.scss';
+class CardList extends Component {
+  render() {
+    const { cardsData} = this.props;
+
+    return (
+      <div className="cardListWrapper">
+        <div className="cardList">
+          {cardsData && cardsData.map(contents => {
```

Comment:
이렇게 줄일 수 있는 꿀팁 하나 드립니다!
```suggestion
          {cardsData?.map(contents => {
```

Reply:
관히님.... 대박입니다!! 감사해요